### PR TITLE
IQ: Add 4 channels

### PR DESCRIPTION
**Pull request by ghost**
[Al-Etejah](https://www.lyngsat.com/tvchannels/iq/Al-Etejah-Satellite-Channel.html)
Al-Jawadain TV
[Al Rasheed TV](https://www.lyngsat.com/tvchannels/iq/Al-Rasheed-TV.html)
[Iraqia Sport](https://www.lyngsat.com/tvchannels/iq/Iraqia-Sport.html) (Should also check the box for Iraqia Sport in #1955)

### CODE REVIEW

**Review by jkemp814 — DISMISSED** [review]
All four streams play well

File: channels/iq.m3u [hunk 1]
```diff
@@ -137,3 +137,11 @@ https://live.karwan.tv/karwan.tv/waar-tv/playlist.m3u8
 https://5a3ed7a72ed4b.streamlock.net/zagrostv/SMIL:myStream.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="ZagrosTV.iq" tvg-country="IQ" tvg-language="Arabic" tvg-logo="" group-title="",Zagros TV (480p) [Not 24/7]
 http://163.172.39.215:25461/live/line/C4@!a3a1@!w72A/48.m3u8
+#EXTINF:-1 tvg-id="AlEtejah.iq" tvg-country="IQ" tvg-language="Arabic" tvg-logo="https://i.imgur.com/jFKKRjP.png" group-title="News",Al-Etejah (576p)
+https://streaming.aletejahtv.iq:1937/etejah-live/smil:etejah.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="AlJawadain.iq" tvg-country="IQ" tvg-language="Arabic" tvg-logo="https://i.imgur.com/tLekUZU.png" group-title="Religious",Al-Jawadain TV (720p)
+https://live.aljawadain.org/live/aljawadaintv/playlist.m3u8
```
Comment (LaneSh4d0w) [inline channels/iq.m3u:143]: 404 not found. Not 24/7 ?

Comment (jkemp814) [reply channels/iq.m3u:143]: Must be. It played just minutes ago. 

Now I get:
https://live.aljawadain.org/live/aljawadaintv/playlist.m3u8: Server returned 404 Not Found

Tried with and without proxychains.

Comment (LaneSh4d0w) [reply channels/iq.m3u:143]: Noted, i'll add a Not 24/7 bracket.

**Review by jkemp814 — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**